arcade/arcade/cli/serve.py

```diff
@@ -124,8 +119,19 @@ def serve_default_worker(
     worker = FastAPIWorker(
         app, secret=worker_secret, disable_auth=disable_auth, otel_meter=otel_handler.get_meter()
     )
+
+    toolkit_tool_counts = {}
     for toolkit in toolkits:
+        prev_tool_count = worker.catalog.get_tool_count()
         worker.register_toolkit(toolkit)
+        new_tool_count = worker.catalog.get_tool_count()
+        toolkit_tool_counts[f"{toolkit.name} ({toolkit.package_name})"] = (
```

Comment:
keeps track of the number of tools added to the catalog (avoids counting disabled tools)